FindAIETools.cmake

```diff
@@ -147,6 +147,26 @@ foreach(comp ${AIETools_FIND_COMPONENTS})
 		set(aieVersionSpecificPath "unknown")
 	endif()
 
+	if(${comp} STREQUAL "AIE")
+		set(aieVersionTargetPath "target")
+	elseif(${comp} STREQUAL "AIE2")
+		set(aieVersionTargetPath "target_aie_ml")
+	elseif(${comp} STREQUAL "AIE2P")
+		set(aieVersionTargetPath "target_aie2p")
+	else()
+		message(ERROR "${comp} not supported")
+		set(aieVersionTargetPath "unknown")
+	endif()
+
+	# Find chesscc
+	find_path(AIETOOLS_${comp}_CHESSCC "chesscc" PATHS ${AIETOOLS_DIR}/tps/lnx64/${aieVersionTargetPath}/bin/LNa64bin
+			NO_DEFAULT_PATH CMAKE_FIND_ROOT_PATH_BOTH)
+	if(NOT AIETOOLS_${comp}_CHESSCC)
+		message(STATUS "Unable to find ${comp} chesscc")
+	else(NOT AIETOOLS_${comp}_CHESSCC)
+		message(STATUS "Found ${comp} chesscc: ${AIETOOLS_${comp}_CHESSCC}")
+	endif(NOT AIETOOLS_${comp}_CHESSCC)
```

Comment:
I'm a bit concerned that this encourages people to call chesscc directly, which is unlikely to be successful.
However, the base path to the bin directly is more likely to be useful.  So maybe export the bin/LNa64bin directory instead?

Reply:
Fixed